student-test/assignment1/Test_validate_username.c

```diff
@@ -18,5 +19,17 @@ void test_validate_my_username()
      * TODO: Replace the line below with your code here as described above to verify your /conf/username.txt 
      * config file and my_username() functions are setup properly
      */
-    TEST_ASSERT_TRUE_MESSAGE(false,"AESD students, please fix me!");
+     char name[100];
+     char test[100];
+     
+    strcpy(name,  malloc_username_from_conf_file());
```

Comment:
@chaisuresh you need to keep a reference to the pointer returned by malloc here, otherwise you will have a memory leak.